lib/commands/keyboard.js

```diff
@@ -34,8 +34,31 @@ export default {
       return false;
     }
   },
+
+  /**
+   * Send keys to the given element or to the application under test.
+   * This API is not supported on tvOS
+   *
+   * @since Xcode 15/iOS 17
+   * @this {import('../driver').XCUITestDriver}
+   * @param {(Key|string)[]} keys Array of keys to type.
+   * Each item could either be a string, that represents a key itself (see
+   * https://developer.apple.com/documentation/xctest/xcuielement/1500604-typekey?language=objc
+   * and https://developer.apple.com/documentation/xctest/xcuikeyboardkey?language=objc)
+   * or a dictionary, if the key should also be entered with modifiers.
+   * @param {string?} [elementId=null] uuid of the element to send keys to.
+   * If the element is not provided then the keys will be sent to the current application.
+   */
+  async mobileKeys(keys, elementId = null) {
+    const url = `/wda/element/${elementId || 0}/keyboardInput`;
+    return await this.wda.proxy.command(url, 'POST', { keys });
```

Comment:
```suggestion
    return await this.proxyCommand(url, 'POST', { keys });
```

Does https://github.com/appium/WebDriverAgent/pull/798 need to handle `0` case in this implementation?

Reply:
yes, my bad